src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/helper/RequestBuilders.kt

```diff
@@ -0,0 +1,178 @@
+package uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.helper
+
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.client.locationsinsideprison.model.Location
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.entity.CvpLinkDetails
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.model.request.AmendVideoBookingRequest
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.model.request.Appointment
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.model.request.AppointmentType
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.model.request.BookingType
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.model.request.CourtHearingType
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.model.request.CreateVideoBookingRequest
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.model.request.PrisonerDetails
+import java.time.LocalDate
+import java.time.LocalTime
+
+class CreateCourtBookingRequestBuilder {
```

Comment:
Created this in an effort to start cleaning up the way requests are created in tests.

I am starting to find the factory methods too hard to follow.